### DISCUSSION TITLE
TP 5 - Ejercicio 2 TATETI ¿Cómo cambiar valor de matriz por un char (X / O)?

### DISCUSSION THREAD

**Discussion by JazminAilenFer** [INGCOM-UNRN-P1/cursada-2023#176] [Preguntas y respuestas]
Me pasa que al asignarle a una posición de la matriz una 'X' o una 'O', me aparece un 88 (que es el código de la X de la tabla ASCII) en cualquier posición. Inicialicé la matriz como `char tablero[FILAS_CTE][COLUMNAS_CTE] = {1, 2, 3, 4, 5, 6, 7, 8, 9};`

**Comment by martinvilu** [comment]
Los números son caracteres, pero no 'visibles', usa el literal apropiado para ver algo como letras como 
`'X'`

**Comment by Nicolino07** [comment]
Puede que el problema sea que estás usando `%d` en el `printf`, deberías usar el `%c`.

**Comment by JazminAilenFer** [comment]
muchas gracias! se me estaba empezando a mezclar todo

**Comment by martinvilu** [comment]
Cuál fue la respuesta que resolvió la pregunta? (Marcela porfis)
